src/new-components/teaser/teaser.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Teaser } from './teaser';
+import ExampleImage from '../../../blog-posts/images/angular.png';
+
+<Meta
+  title="Components/Teaser"
+  component={{ Teaser }} />
+
+# Teaser
+Teasers are used to link to another page and give the user a brief overview of the content of that page.
+This requires a `title` and a short text, which is entered as a child.
+In addition, an `illustration` or an `image`, a formatted date (`dateFormatted`), a link to another page (`linkTo`) and a `topline` can be displayed.
+You can't pass in an image **and** illustration. If you pass in an image and illustration the illustration will be taken.
+
+<ArgsTable of={Teaser} />
```

Comment:
That looks great and useful in the introduction 👍
![Screen Shot 2021-12-14 at 14 48 44](https://user-images.githubusercontent.com/1701755/146010800-307eb451-fc33-4450-b3e6-087e5e8b4bb1.png)